packages/language-server/test/unit/quick-fix.test.ts

```diff
@@ -33,6 +35,11 @@ describe("The @ui5-language-assistant/language-server diagnostics quick fix func
     appContext.manifestDetails.flexEnabled = true;
   });
 
+  afterEach(() => {
+    appContext = getDefaultContext(ui5SemanticModel);
```

Comment:
This seems a bit problematic and error-prone, that tests leave a dirty environment after execution.

Reply:
well. This is because we do manipulate `appContext` see line 91 
```typescript
appContext.viewFiles[appContext.documentPath] = ast;
```
after each is basically clean up that dirty env. We could avoid this by using `befreEach` instead of `beforeAll` but this `generateModel` seems to be a heavy operation.